server/proxy/pf_channels.c

```diff
@@ -26,7 +26,9 @@
 #include <freerdp/gdi/gfx.h>
 
 #include <freerdp/client/rdpei.h>
+#if defined(CHANNEL_TSMF_CLIENT)
```

Comment:
I'd just remove the include, there's no use of TSMF in proxy and I don't see a reason there will ever be.